driver.go

```diff
@@ -28,15 +29,14 @@ const (
 
 var (
 	defaultTritonAccount = ""
-	defaultTritonKeyPath = os.Getenv("HOME") + "/.ssh/id_rsa"
+	defaultTritonKeyPath = "" // os.Getenv("HOME") + "/.ssh/id_rsa"
```

Comment:
Oh nice, does this default now surface in `triton-go` itself?

Reply:
By default `triton-go` falls back to pulling your `SDC_KEY_ID` key out of your `SSH_AUTH_SOCK` for `ssh-agent` based key signing. We're actually doing this manually [a few lines down](https://github.com/joyent/docker-machine-driver-triton/pull/10/files#diff-749da71b40f8ff06fc9e78ce917b0cceR147).

So I didn't see the need to force the key path as being required in order to use this tool. It _should_ still be backwards compatible with the old command-line flag.